Themis/Views/Assessment/Modeling Exercise/UMLRenderer.swift

```diff
@@ -24,14 +25,17 @@ struct UMLRenderer: View {
     
     var body: some View {
         ZStack(alignment: .topLeading) {
-            Image("umlRendererBackground")
-                .resizable(resizingMode: .tile)
-            
+            /// Render the Grid Background from the Apollon Package
+            ApollonViewViewModel.getGridBackground()
```

Comment:
I'd create a separate view (something like `ApollonGridBackgroundView`) and use that instead of a static function. But you can still keep it like this because I'm also not sure if there's any disadvantage of using static functions to generate views :D

Reply:
Yeah, it's probably not the best practice😆 I changed the structure of the Apollon module and created a `GridBackgroundView()`, which Themis can now access.